int16 speedLeft
int16 speedRight
int16 duration
---
string result